content/en/docs/KRE/getting_started/index.md

```diff
@@ -4,24 +4,20 @@ description: A quick guide to start working with KRE.
 weight: 5
 ---
 
-## Pre requisites
-
-In order to work with KRE you will need these requisites. This is not completely mandatory but if you read all docs before starting, it would help to understand better what you are doing on each step.
-
-- [Basic helm knowledge]({{< relref "docs/KRE/installation/helm" >}})
-- [KRE basic concepts]({{< relref "docs/KRE/concepts" >}})
-
-
 ## Installation
 
-To start working with KRE you first need to get it installed in some Kubernetes cluster. You can choose different installation targets like [local Minikube]({{< relref "docs/KRE/installation/minikube" >}}), [EKS]({{< relref "docs/KRE/installation/EKS" >}}) or [GKE]({{< relref "docs/KRE/installation/GKE" >}}). 
+The best way to start working with KRE, to test it and understand how it works, is deploy it on your local machine and follow a basic instructions to create an example project.
```

Comment:
change:
-  is deploy for is **to** deploy and a basic instructions for basic instructions '